### PR TITLE
User requests

### PR DESCRIPTION
**Pull request by suzkiee**
# Description
This PR adds following logic for the back end API: 
- Chores: index and show
- Roommates: index, show, create 
- Updates Factory_bot factories and configuration
- Updates rubocop.yml file 
- Adds respective tests and serializers 

#Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Closes # (issue)

# How Has This Been Tested?
- [ ] This has not been tested
- [x] This has been tested

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/controllers/api/v1/chores_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  def show
+    chore = Chore.find(params[:id])
+    render json: ChoresSerializer.new(chore), status: 200
+
+  rescue ActiveRecord::RecordNotFound
+    render json: {
+      message: 'Not Found',
+      errors: ["Could not find chore with id##{params[:id]}"]
+    }, status: :not_found
```
Comment (Aphilosopher30) [inline app/controllers/api/v1/chores_controller.rb:16]: status should be a number like 404

File: app/services/map_quest_service.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class MapQuestService
```
Comment (Aphilosopher30) [inline app/services/map_quest_service.rb:1]: location service would be better name

File: app/controllers/api/v1/roommates_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    render json: RoommatesSerializer.new(user), status: 200
+
+  rescue ActiveRecord::RecordNotFound
+    render json: {
+      message: 'Not Found',
+      errors: ["Could not find user with id##{params[:id]}"]
+    }, status: :not_found
+  end
+
+  def create
```
Comment (EmmyMorris) [inline app/controllers/api/v1/roommates_controller.rb:19]: Great job with this create method!🤩

File: app/models/chore.rb [hunk 4]
```diff
@@ -1,7 +1,9 @@
 class Chore < ApplicationRecord
   validates :task_name, :description, :outdoor, presence: true
-  validates :weight, presence: true, numericality: true
-  validates :frequency, presence: true, numericality: true
+  validates :weight, :frequency, presence: true, numericality: true
+  # enum frequency: {1: "weekly", 2: "bimonthly", 3: "monthly"}
+  enum frequency: {weekly: 1, bimonthly: 2, monthly: 3}
```
Comment (EmmyMorris) [inline app/models/chore.rb:5]: I really these enum options for frequency❗️

File: app/services/open_weather_service.rb [hunk 5]
```diff
@@ -1,7 +1,7 @@
 class OpenWeatherService
 
   def self.get_forecast_data(coordinates)
-    new.request_api("/data/2.5/onecall?lat=#{coordinates[:lat]}&lon=#{coordinates[:lon]}&exclude={part}")
+    new.request_api("/data/2.5/onecall?lat=#{coordinates[:lat]}&lon=#{coordinates[:lon]}&exclude=minutely,hourly,alerts")
```
Comment (EmmyMorris) [inline app/services/open_weather_service.rb:4]: Amazing job figuring this out!

File: db/migrate/20210729012612_add_columns_to_roommates.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+class AddColumnsToRoommates < ActiveRecord::Migration[5.2]
```
Comment (EmmyMorris) [inline db/migrate/20210729012612_add_columns_to_roommates.rb:1]: Amazing job handling all of the migrations!🥳   🙌

File: spec/rails_helper.rb [hunk 7]
```diff
@@ -7,7 +7,7 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
-
+require 'factory_bot'
```
Comment (EmmyMorris) [inline spec/rails_helper.rb:10]: 🏭 Nice work creating all factories! 🏭.

**Review by EmmyMorris — APPROVED** [review]
🎊🎉I think everything looks great! Great job making all of the migrations, factories and tests! 🎉🎊

**Review by denverdevelopments — APPROVED** [review]

**Review by ztrokey — APPROVED** [review]
Looks great!

**Pull request merged**